src/qt/veil.cpp

```diff
@@ -649,12 +649,15 @@ int main(int argc, char *argv[])
         return EXIT_FAILURE;
     }
 
-    // Preferences check
-    // If the UI settings have a different value than the server args then use the UI settings.
+    // Automint denom
+    // If nautomintdenom is set, use it
+    // Else use saved UI settings
```

Comment:
This change is a behavior change (changing the preference from the UI setting to the veil.conf setting) so definitely needs a statement in the release notes; as those that have both will see a change in behavior.